site/blog/series-a-announcement.md

```diff
@@ -26,7 +26,7 @@ tags: [company-update]
 
 Promptfoo has raised an $18.4 million Series A led by Insight Partners, with participation from Andreessen Horowitz.
 
-We started Promptfoo in 2024 after watching AI systems grow increasingly complex while security tooling lagged behind. Today, over 125,000 developers use our open source framework and more than 30 Fortune 500 companies run our platform in production.
+We started Promptfoo in 2024 after watching AI systems grow increasingly complex while security tooling lagged behind. Today, over 200,000 developers use our open source framework and more than 30 Fortune 500 companies run our platform in production.
```

Comment:
Update the F500 count as well?

Reply:
should not have updated this document at all it's point in time